.github/workflows/build.yml

```diff
@@ -15,17 +15,17 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.23.0
```

Comment:
Update Go version referenced in README file.

Reply:
will do